test/unittests/instantiate_test.cpp

```diff
@@ -53,12 +52,8 @@ TEST(instantiate, imported_functions_multiple)
         "0061736d0100000001090260017f017f600000021702036d6f6404666f6f310000036d6f6404666f6f320001");
     const auto module = parse(bin);
 
-    auto host_foo1 = [](Instance&, span<const uint64_t>, int) -> execution_result {
-        return {true, {0}};
-    };
-    auto host_foo2 = [](Instance&, span<const uint64_t>, int) -> execution_result {
-        return {true, {}};
-    };
+    auto host_foo1 = [](Instance&, span<const uint64_t>, int) -> execution_result { return Void; };
```

Comment:
These are not executed, but `foo1` import returns a result (`foo2` is void)

Reply:
But both were also traps in the same time. You cannot have trap + value now.

Reply:
Before: both were trapping, but `foo1` supposed to return a result and `foo2` not
After: `foo1` does not trap and returns nothing, `foo2` traps

@gumb0 is this the correct intended behaviour?

Reply:
The implementation doesn't matter, I would just make them both trap.